src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -270,175 +195,4 @@ describe('<FilesPage>', () => {
     );
     expect(tree.props.children).to.be.null;
   });
-
-  it('should hide requested files when closed', () => {
-    const claim = {
-      attributes: {
-        open: false,
-        eventsTimeline: [],
-      },
-    };
-    const tree = SkinDeep.shallowRender(<FilesPage claim={claim} />);
-
-    expect(tree.subTree('RequestedFilesInfo')).to.be.false;
-  });
-
-  it('should show requested files when open', () => {
```

Comment:
I think we want this test we just need to update the claim for lighthouse

Reply:
Done